Topic 0/src/Main/Main.java

```diff
@@ -0,0 +1,17 @@
+package Main;
+
+import Classes.Builder;
+import Classes.Home;
+import Classes.Hospital;
+
+public class Main {
+
+	public static void main(String[] args) {
+		Builder hospital = new Hospital(30, 50, "Acevedo 1234", "2234567689", 14, 12, "San Cayetano");
```

Comment:
You have not implemented the Builder pattern here. In this case, the builder class is just an abstraction of building, with common building attributes.
Also, in this case all attributes are required (you can't create a building without phone for example).

Please read this example: http://www.avajava.com/tutorials/lessons/builder-pattern.html and identify the different components.

Let me know if you have any question before starting with implementation.